pages/resources.tsx

```diff
@@ -29,6 +29,7 @@ import { useFormik } from 'formik'
 import Masonry, { ResponsiveMasonry } from 'react-responsive-masonry'
 import { t } from 'utils/i18n'
 import { filterResources } from 'utils/filter-resources'
+import NextLink from 'next/link'
```

Comment:
Move import over local utils imports pls :)

Reply:
Tbh I can not figure out any structure/order in the imports of the whole repo (until now regarding the utils after your comment). I just let my `"source.organizeImports"` run over that file to sort them alphabetical. Tried to avoid this to do not that much changes within the imports. 

Hope this is fine. :smiley:

Reply:
@m4x3d there's an eslint plugin that does that: https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md. Try adding this config in this PR, and then we'll make another issue to fix the remaining cases where we have mixed imports.

Reply:
@nikolovlazar I've open a PR where import ordering is included aswell as some other basic linting https://github.com/chakra-ui/chakra-ui-docs/pull/350

Reply:
@noobinthisgame just saw it now. I'm already reviewing it. Thank you!

Reply:
@nikolovlazar Talked about it with @noobinthisgame after my comment. We can let this PR open until his is merged, guess this was your thought too.